docker/init.sql

```diff
@@ -0,0 +1 @@
+-- create database coupon if not exists coupon;
```

Comment:
![image](https://github.com/user-attachments/assets/27a1ebfd-f3ae-4e57-a442-0a4d2ee3a441)
`coupon` 데이터베이스가 없어서 실행이 실패하고 있어요 ㅠㅠ
이왕 init.sql 파일 볼류마이징 한 김에 여기서 생성을 해주는 게 좋을 것 같아요!
주석을 없애고 아래와 같이 수정해주면 데이터베이스가 생성된답니다!
그리고 이 방법 외에도 다른 방법들도 있으니 찾아보시고 원하는 방법대로 적용해주세요!

```suggestion
create database if not exists coupon;
```

Reply:
주석해놓고 이것저것 실험해보다가 그대로 푸쉬했네요 😭
주석 해제했습니다